src/AzureRMTools.ts

```diff
@@ -28,7 +28,7 @@ import { UnrecognizedFunctionIssue } from "./UnrecognizedFunctionIssue";
 export async function activateInternal(context: vscode.ExtensionContext, perfStats: { loadStartTime: number; loadEndTime: number }): Promise<void> {
     ext.context = context;
     ext.reporter = createTelemetryReporter(context);
-    ext.outputChannel = vscode.window.createOutputChannel("Azure Resource Manager Tools");
+    ext.outputChannel = vscode.window.createOutputChannel("ARM Tools");
```

Comment:
I feel @EricJizbaMSFT has a valid point: `ARM` doesn't show up any azure related pages on Bing or Google search, but Azure ARM does. `Azure Resource Manager Tools` seems more explanatory. If it's too long a name, `Azure Tools - ARM` might be less ambiguous.

Q: Is `ARM` a branding Azure uses externally? I've seen MSFT employees use it, and as abbreviations inside Azure Resource Manager pages. Maybe @bmoore-msft can answer this.

Reply:
This came from legal, see forwarded e-mail.

Reply:
The email still says "ARM" is discouraged. Why can't we just use `Azure Resource Manager` everywhere and follow recommended best practices? It's not _THAT_ long. Also we don't have `Tools` in our other extension names, which will help shorten it

Reply:
It gets rather unwieldy in some places, like the language name "Azure Resource Manager Deployment Template", "Azure Resource Manager Language Server", and as the source for diagnostic issues.

Reply:
The point about "tools" is good.  I guess we could just call it "Azure Resource Manager".  Maybe "Azure Resource Manager Templates"??

Reply:
"ARM" is in formal - in tooling/docs we should be using "Azure Resource Manager".  In GitHub PR comments we can use ARM all we want ;)